test/unit/event.js

```diff
@@ -2678,6 +2678,43 @@ test( ".off() removes the expando when there's no more data", function() {
 	}
 });
 
+test( "preventDefault() on focusin does not throw exception", function( assert ) {
+	expect( 1 );
+
+	var done = assert.async(),
+		input = jQuery( "<input>" ).appendTo( "#form" );
```

Comment:
Agreed for sanitation and sanity reasons, although this goes through the `rsingleTag` path anyway.